src/renderer/viz/expressions/SVG.js

```diff
@@ -4,6 +4,7 @@ import { checkMaxArguments } from './utils';
 export default class SVG extends Image {
     constructor (svg) {
         checkMaxArguments(arguments, 1, 'svg');
+        // This doesn't work in Firefox for local SVG images
```

Comment:
This is fine for our internal reference, but we should probably replicate this note in public jsdoc

Reply:
This expression is not documented, we're using it internally. If we want to make it public in the future we can, taking into account that it will imply a minor release.